torch/testing/_internal/common_device_type.py

```diff
@@ -384,6 +447,22 @@ def instantiate_device_type_tests(generic_test_class, scope, except_for=None, on
         device_type_test_class.__module__ = generic_test_class.__module__
         scope[class_name] = device_type_test_class
 
+# Decorator that defines the ops a test should be run with
+# The test signature must be:
+#   <test_name>(self, device, dtype, op_meta)
+# For example:
+# @ops(unary_ufuncs)
+# test_numerics(self, device, dtype, op_meta):
+#   <test_code>
+class ops(object):
+    def __init__(self, op_list, *, unsupported_dtypes_only=False):
```

Comment:
`unsupported_types_only` seems super specific for the particular example test you were looking at. I'm half expecting you to quickly start adding things like `unsupported_devices_only` and `unsupported_strides_only`, etc., etc. Is that the intention? (I.e., that we're going to have a ton of available kwargs here?)

Reply:
I don't think so. The test that uses it could also be written without this kwarg by enumerating the dtypes within the test, instead, I just thought this was a little more straightforward. Either way is fine.